src/fides/data/sample_project/docker-compose.yml

```diff
@@ -23,20 +22,8 @@ services:
     environment:
       # TODO: remove once merged: https://github.com/ethyca/fides/pull/1552
       FIDES__EXECUTION__WORKER_ENABLED: "True"
-      # FIDES__CLI__SERVER_HOST: "fides"
-      # FIDES__CLI__SERVER_PORT: "8080"
-
-      # FIDES__DATABASE__USER: "postgres"
-      # FIDES__DATABASE__PASSWORD: "fides"
-      # FIDES__DATABASE__DB: "fides"
-      # FIDES__DATABASE__SERVER: "fides-db"
-      # FIDES__DATABASE__PORT: "5432"
-
-      # FIDES__REDIS__ENABLED: "True"
-      # FIDES__USER__ANALYTICS_OPT_OUT: "True"
-    volumes:
-      - ./.fides:/fides/.fides
-
+      # Hardcode the path to the sample config file
+      FIDES__CONFIG_PATH: "/fides/src/fides/data/sample_project/fides.toml"
```

Comment:
Right; you're doing this because mounting `.fides/` isn't going to work when running this from an arbitrary directory outside the repo (e.g. `$HOME`)?

Reply:
yep, absolute path is safer here and we actually only care about the `toml` file (for now), since we expect the user to be able to load their own files from the local CLI